src/stories/Input.stories.tsx

```diff
@@ -32,6 +32,68 @@ function Template(args: any) {
   )
 }
 
+function CustomInputTemplate() {
+  return (
+    <Flex
+      direction="column"
+      maxWidth="400px"
+    >
+      <ExtendTheme
+        theme={{
+          // Theme extension to give the Input a custom end icon
+          Input: {
+            Root: [
+              {
+                paddingRight: 0,
+              },
+            ],
+            EndIcon: [
+              {
+                alignSelf: 'stretch',
+                paddingHorizontal: 'small',
+                backgroundColor: 'fill-two',
+              },
+            ],
+          },
+        }}
+      >
+        <Div>
+          <Input
+            large
+            width="100%"
+            placeholder="my-company"
+            endIcon={(<>.onplural.sh</>)}
+          />
+        </Div>
+        <Div marginTop="medium">
+          <Input
+            width="100%"
+            placeholder="my-company"
+            endIcon={(
```

Comment:
this isn't a huge must-have but I think we should support suffix and prefix string variables to append to inputs, since this is a pretty common design pattern.

Reply:
Agreed. I've modified this PR to now export a wrapped version of the honorable Input component that allows for setting prefixes and suffixes.

Reply:
It _should_ just be a drop-in replacement that doesn't change any current usage if you're not using 'suffix' or 'prefix' props.